src/main/scala/nl.knaw.dans.easy.solr4files/components/AuthorisationItem.scala

```diff
@@ -32,7 +32,9 @@ case class AuthorisationItem(itemId: String,
                              owner: String,
                              dateAvailable: DateTime,
                              accessibleTo: RightsFor.Value,
-                             visibleTo: RightsFor.Value
+                             visibleTo: RightsFor.Value,
+                             licenseKey: String,
+                             licenseTitle: String
```

Comment:
one comma makes the next diff look much more beautiful 😜 

```suggestion
                             licenseTitle: String,
```